exercises/pov/.meta/template.j2

```diff
@@ -0,0 +1,55 @@
+{%- import "generator_macros.j2" as macros with context -%}
+
+{%- macro test_case(case) %}
+    def test_{{ case["description"] | to_snake }}(self):
+        tree = {{ write_tree(case["input"]["tree"]) }}
+        {% if case["property"] == "fromPov" -%}
+        {{ test_from_pov(case) }}
+        {%- elif case["property"] == "pathTo" -%}
+        {{ test_path_to(case) }}
+        {%- endif -%}
+{% endmacro -%}
+
+{%- macro test_from_pov(case) -%}
+        {% if case["expected"] -%}
+        expected = {{ write_tree(case["expected"]) }}
+        self.assertTreeEquals(tree.{{ case["property"] | to_snake }}("{{ case["input"]["from"] }}"), expected)
+        {% else -%}
+        with self.assertRaisesWithMessage(ValueError):
+            tree.{{ case["property"] | to_snake }}("{{ case["input"]["from"] }}")
+        {% endif -%}
+{% endmacro -%}
+
+{%- macro test_path_to(case) -%}
+        {% if case["expected"] -%}
```

Comment:
```suggestion
        {% if not case is error_case -%}
```

Reply:
The canonical data for the pov exercise seems to indicate an error case by having the "expected" value be `null`. This means `error_case` will not work as expected. 

Should this be changed in the canonical data, or should we just work around it in the template and test file?

Reply:
Hmm... this seems to have been a matter of interpretation by the original author of this exercise in the Python track. For now, keep your changes as they are, and we can revisit whether this behavior in the tests should be changed at a later date.